sherlock_project/resources/data.json

```diff
@@ -613,6 +613,13 @@
     "urlMain": "https://www.dailymotion.com/",
     "username_claimed": "blue"
   },
+  "DailyDev":{
+    "errorType": "message",
+    "errorMsg":"Page not found.",
+    "url": "https://app.daily.dev/{}",
+    "urlMain": "https://app.daily.dev/",
+    "username_claimed": "nj05"
+  },
```

Comment:
Returns positive for all queries
`errorType` of `message` should be a last resort behind status_code due to l10n issues